classes/PHPTAL/Php/Attribute/TAL/Attributes.php

```diff
@@ -46,6 +46,8 @@ class PHPTAL_Php_Attribute_TAL_Attributes
      * value for default keyword
      */
     private $_default_escaped;
+    private string $_attribute;
+    private string $_attkey;
```

Comment:
These need to have a default value, as they are not initialized in `__construct`